package.json

```diff
@@ -16,9 +16,9 @@
     "mocha": "^3.5.0"
   },
   "scripts": {
-    "test": "mocha && npm run lint",
-    "test:watch": "mocha -w",
-    "lint": "$(npm bin)/jshint src/ test/"
+    "test": "mocha",
+    "test:watch": "mocha -w -R dot",
+    "lint": "jshint src/ test/"
```

Comment:
does this works when jshint is not installed globally?

Reply:
Yes. See https://docs.npmjs.com/misc/scripts#path.

BTW, it's the same as in "mocha", isn't it?

Reply:
right! good to know